circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  services:
+    - docker
+
+test:
+  override:
+    - echo 0
+
+database:
+  override:
+    - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+
+deployment:
+  package:
+    branch: /.*/
+    commands:
+      - docker build -t keen/twemproxy:$CIRCLE_BUILD_NUM .
```

Comment:
Build the image in the `dependencies` section so that it can be started in the `database` section and then tested in the `test` section.